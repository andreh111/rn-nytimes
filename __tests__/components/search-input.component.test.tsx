import React from 'react';
import renderer from 'react-test-renderer';

import SearchInput from '../../src/components/search-input/search-input.component';

describe('search-input.component test', () => {
  it('search-input.component should match snapshot', () => {
    const component = renderer.create(<SearchInput
          query='python' 
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
