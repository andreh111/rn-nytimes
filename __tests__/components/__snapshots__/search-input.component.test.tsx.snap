// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search-input.component test search-input.component should match snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "borderColor": "#585858",
      "borderRadius": 10,
      "borderWidth": 1,
      "flexDirection": "row",
      "height": 50,
      "justifyContent": "space-between",
      "margin": 10,
      "padding": 5,
    }
  }
>
  <TextInput
    placeholder="Search..."
    style={
      Object {
        "flex": 1,
        "fontSize": 18,
      }
    }
    testID="search-input"
    value="python"
  />
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": "#585858",
          "fontSize": 25,
        },
        undefined,
        Object {
          "fontFamily": "Ionicons",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    ï”…
  </Text>
</View>
`;
